### PR TITLE
Remove CMS examples from Data Fetching

### LINKED ISSUE #2577: Consolidate CMS docs from Data Fetching
The Data Fetching guide includes content that predates our dedicated CMS guides section: https://docs.astro.build/en/guides/data-fetching/#fetch-from-a-headless-cms. We should consolidate this.

```[tasklist]
- [x] Remove Storyblok & WordPress examples (maybe double check if it makes sense to move these into the CMS guides, but those are both fairly extensive IIRC)
- [x] Create new Crystallize CMS guide stub based on the Data Fetching example
- [x] Leave top-level “Fetch from a Headless CMS” heading but make its content a short & sweet affirmation that this is possible with links to the CMS guides
```

### PR DESCRIPTION
**Pull request by Jutanium**
#### What kind of changes does this PR include?

- New or updated content

#### Description

- Closes #2577
- Adds Crystallize stub with example from Data Fetching
- Removes examples from Data Fetching
- Links from Data Fetching to CMS Guides
- Remove redundant leading exclamation sentence

### CODE REVIEW

File: src/pages/en/guides/data-fetching.mdx
Comment (sarah11918) [inline src/pages/en/guides/data-fetching.mdx:111]:
```suggestion
See our [CMS Guides](/en/guides/cms/) for full details on integrating Astro with headless CMSes including Storyblok, Contentful, and WordPress.
```
link fix

**Comment by Jutanium** [comment]
I edit the SVG to just contain the crystal but I did not do any optimizing; I'll wait for @delucis's approval!

File: public/logos/crystallize.svg
Comment (delucis) [inline public/logos/crystallize.svg:6]: I usually run SVGs through some reasonably aggressive settings on https://jakearchibald.github.io/svgomg/

Here’s the result for this one:

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 57 85">
  <path fill="#BFF6F8" d="M45.7 37.4 33.4 85 17.6 74.8 29.8.3l18 29.1-2.1 8Z"/>
  <path fill="#DFFAFB" d="M11.7 38.8 7.5 68.4 5 66.9.4 32.5l11.3 6.3Z"/>
  <path fill="#CFF8FA" d="m25.1 19.2-9 54.7L9 69.3l8.8-61.5 7.4 11.4Z"/>
  <path fill="#ACF3F6" d="M57.5 35.3 47.3 74.4 40.6 80l-5.4 4.3L46.7 40l10.8-4.6Z"/>
</svg>
```

329 bytes → 264 bytes (80.24%) gzipped

This was already pretty small so it’s not a big deal but can be a good flow to catch any SVGs that aren’t (sometimes they contain random junk data from a software export for example).

**Review by delucis — APPROVED** [review]
This is great! Thanks, Dan. Suggested compressing the SVG slightly but otherwise spot on 🙌

**Pull request merged**
